fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/temp/expressions/ExplodeExpression.java

```diff
@@ -51,16 +48,10 @@
 @API(API.Status.EXPERIMENTAL)
 public class ExplodeExpression implements RelationalExpression, InternalPlannerGraphRewritable {
     @Nonnull
-    private final CorrelationIdentifier correlationIdentifier;
-    @Nonnull
-    private final List<String> fieldNames;
-    @Nonnull
-    private final FieldValue resultValue;
+    private final Value resultValue;
 
-    public ExplodeExpression(@Nonnull final CorrelationIdentifier correlationIdentifier, @Nonnull final List<String> fieldNames) {
-        this.correlationIdentifier = correlationIdentifier;
-        this.fieldNames = fieldNames;
-        this.resultValue = new FieldValue(QuantifiedColumnValue.of(correlationIdentifier, 0), fieldNames);
```

Comment:
I don't see any reason not to keep this constructor and have it call the new one; there are three callers essentially inlining this code now.

Reply:
I rather make a static factory method.